packages/tooltips/examples/basic.md

```diff
@@ -0,0 +1,189 @@
+    return longValue;
+  }
+
+  return shortValue;
+};
+
+<State
+  initialState={{
+    placement: 'top',
+    size: 'default',
+    hasArrow: true,
+    delayMS: 500,
+    isVisible: true
+  }}
+>
+  {(state, setState) => (
+    <Grid>
+      <Row>
+        <Col md={5}>
+          <Well recessed>
+            <StyledSpacer>
+              <Dropdown
+                selectedItem={state.placement}
+                onSelect={placement => setState({ placement })}
+              >
+                <SelectField>
+                  <SelectLabel>Placement</SelectLabel>
+                  <Select small>{state.placement}</Select>
+                </SelectField>
+                <Menu small>
+                  <Item value="auto">auto</Item>
+                  <Item value="top">top</Item>
+                  <Item value="top-start">top-start</Item>
+                  <Item value="top-end">top-end</Item>
+                  <Item value="bottom">bottom</Item>
+                  <Item value="bottom-start">bottom-start</Item>
+                  <Item value="bottom-end">bottom-end</Item>
+                  <Item value="end">end</Item>
+                  <Item value="end-top">end-top</Item>
+                  <Item value="end-bottom">end-bottom</Item>
+                  <Item value="start">start</Item>
+                  <Item value="start-top">start-top</Item>
+                  <Item value="start-bottom">start-bottom</Item>
+                </Menu>
+              </Dropdown>
+            </StyledSpacer>
+            <StyledSpacer>
+              <Dropdown selectedItem={state.size} onSelect={size => setState({ size })}>
+                <SelectField>
+                  <SelectLabel>Size</SelectLabel>
+                  <Select small>{state.size}</Select>
+                </SelectField>
+                <Menu small>
+                  <Item value="default">default</Item>
+                  <Item value="small">small</Item>
+                  <Item value="medium">medium</Item>
+                  <Item value="large">large</Item>
+                  <Item value="extra-large">extra-large</Item>
+                </Menu>
+              </Dropdown>
+            </StyledSpacer>
+            <StyledSpacer>
+              <Field>
+                <Toggle
+                  checked={state.hasArrow}
+                  onChange={e => setState({ hasArrow: e.target.checked })}
+                >
+                  <Label>Arrow</Label>
+                </Toggle>
+              </Field>
+            </StyledSpacer>
+            <StyledSpacer>
+              <Field>
+                <Toggle
+                  checked={state.isVisible}
+                  onChange={e => setState({ isVisible: e.target.checked })}
+                >
+                  <Label>Force visibility</Label>
+                </Toggle>
+              </Field>
+            </StyledSpacer>
+            <StyledSpacer>
+              <Field>
+                <Label>Delay ({state.delayMS} ms)</Label>
```

Comment:
I noticed an a11y with this element while running Axe / Lighthouse.

<img width="1128" alt="Screen Shot 2019-11-21 at 9 52 55 AM" src="https://user-images.githubusercontent.com/1811365/69363159-ce7f9a80-0c44-11ea-990f-7faa17ce92e6.png">

Reply:
This is an axe false-positive relating to the `useField()` hook. This doesn't cause any errors for audits or screenreaders.

Reply:
As a side note, I do think our `useField` hook is a bit brute force and could use some finesse in the future. There are other semantic configurations that aren't handled very elegantly (i.e. Label/Hint over multiple radio inputs).